pages/post.js

```diff
@@ -0,0 +1,15 @@
+import { useRouter } from "next/router";
+import Layout from "../comps/MyLayout";
+
+const Page = () => {
+  const router = useRouter();
+
+  return (
```

Comment:
Basically yeah, you have to _explicitly_ return here since you're declaring a `const` in the function body and can't _implicitly_ return (which is what it's technically called when you do like `() => <div />`)